pom.xml

```diff
@@ -217,6 +217,7 @@
     <eclipse.jetty.version>9.4.54.v20240208</eclipse.jetty.version>
     <nimbus-jose-jwt.version>9.37.3</nimbus-jose-jwt.version>
     <bouncycastle.version>1.77</bouncycastle.version>
+    <pulsar.version>2.11.4</pulsar.version>
```

Comment:
The dependency is not declared in the root pom though. Should we consider moving it back?